.drone.env

```diff
@@ -1,7 +1,3 @@
 # The version of OCIS to use in pipelines that test against OCIS
 OCIS_COMMITID=b2c2416ae13e17b7bb5f2cc4e90459e79f4ecb2b
 OCIS_BRANCH=master
-
-# The test runner source for API tests
```

Comment:
Note: this comment was not correct - this repo does not run core API tests. 
`CORE_COMMITID` was actually used to specify the commit id in core from which to install oC10.